openapi/adintel/adintel.yaml

```diff
@@ -192,6 +287,47 @@ components:
         status: trustedTester
       x-tags:
         - accountStats
+    connectedAccount:
+      title: connectedAccount
+      type: object
+      description: 'An advertising account that’s connected within Advertising Intelligence from integrated campaign management platforms, including Google Adwords, Facebook Ads and Local Ads.'
+      x-examples: {}
+      properties:
+        id:
+          type: string
+          description: The resource ID of the Advertising Account
+        type:
+          type: string
+          description: connectedAccount
+        attributes:
+          type:
+            - object
+          properties:
+            name:
+              type: string
+              description: The name of the ad account
+            currencyCode:
+              type: string
+              description: "\t\nThe currency code that the ad account uses"
+            credentialInvalid:
+              type: boolean
+              description: 'Whether or not if the saved credential for the ad account is invalid, e.g. user no longer have access to the account, or user revoked access.'
+            provider:
+              type: string
+              description: The Advertising provider the account is from
+            created:
+              type: string
+              description: The date time that the ad account is connected
+            updated:
+              type: string
+              description: The date time that the entity was last updated
+      required:
+        - id
+        - type
+      x-tags:
+        - connectedAccount
+      x-lifecycle:
+        status: trustedTester
```

Comment:
We will go with a status of `proposed` until after we are done testing our server code.